repo/packages/C/cockroachdb/0/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "description": "CockroachDB on DC/OS",
+  "framework": true,
+  "maintainer": "alex@cockroachlabs.com",
+  "minDcosReleaseVersion": "1.10",
+  "name": "cockroachdb",
+  "packagingVersion": "3.0",
+  "postInstallNotes": "CockroachDB is being installed!\n\n\tDocumentation: https://www.cockroachlabs.com/docs\n\tIssues: contact https://www.cockroachlabs.com/docs/support-resources.html\n\tRelease Notes:https://www.cockroachlabs.com/docs/v1.0.2.html",
+  "postUninstallNotes": "CockroachDB has been uninstalled.",
+  "preInstallNotes": "This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Preview packages should never be used in production!",
+  "selected": false,
+  "tags": [
+    "cockroachdb",
+    "sql",
+    "database"
+  ],
+  "version": "1.0.1"
```

Comment:
`version` should be <integration-version>-<base-tech-version>. So if cockroachdb version is 1.2.3 then, -> 1.0.1-1.2.3